app/_src/production/cp-deployment/multi-zone.md

```diff
@@ -66,6 +82,81 @@ The global control plane on Kubernetes must reside on its own Kubernetes cluster
     By default, it's exposed on {% if_version lte:2.1.x %}[port 5685](/docs/{{ page.version }}/networking/networking){% endif_version %}{% if_version gte:2.2.x %}[port 5685](/docs/{{ page.version }}/production/deployment/networking/){% endif_version %}. In this example the value is `35.226.196.103:5685`. You pass this as the value of `<global-kds-address>` when you set up the zone control planes.
 
 {% endtab %}
+
+{% if_version gte:2.2.x %}
+{% tab global-control-plane Universal on Kubernetes using Helm %}
+
+1. Set `controlPlane.environment=universal` and `controlPlane.mode=global` in the chart (`values.yaml`).
+
+1. Set `controlPlane.secrets` with database sensitive information
+
+   ```yaml
+   # ...
+       secrets:
+         postgresDb:
+           Secret: your-secret-name
+           Key: POSTGRES_DB
+           Env: KUMA_STORE_POSTGRES_DB_NAME
+         postgresHost:
+           Secret: your-secret-name
+           Key: POSTGRES_HOST_RW
+           Env: KUMA_STORE_POSTGRES_HOST
+         postgrestUser:
+           Secret: your-secret-name
+           Key: POSTGRES_USER
+           Env: KUMA_STORE_POSTGRES_USER
+         postgresPassword:
+           Secret: your-secret-name
+           Key: POSTGRES_PASSWORD
+           Env: KUMA_STORE_POSTGRES_PASSWORD
+   ```
```

Comment:
would be cool to also add a template of Secret. This might not be super clear that `Key` is a key in the secret data also what is the type of secret.
```
apiVersion: v1
kind: Secret
metadata:
  name: your-secret-name
type: Opaque
data:
  POSTGRES_DB: ...
  POSTGRES_HOST_RW: ...
  KUMA_STORE_POSTGRES_USER: ...
  KUMA_STORE_POSTGRES_PASSWORD: ...
```